analyzers/nodejs/nodejs_test.go

```diff
@@ -1,7 +1,82 @@
 package nodejs_test
 
-import "testing"
+import (
+	"os"
+	"testing"
+
+	"github.com/fossas/fossa-cli/files"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/fossas/fossa-cli/analyzers/nodejs"
+	"github.com/fossas/fossa-cli/buildtools/npm"
+	"github.com/fossas/fossa-cli/module"
+	"github.com/fossas/fossa-cli/pkg"
+)
 
 func TestNoDependencies(t *testing.T) {
 	t.Skip("unimplemented")
 }
+
+type MockNPM struct{}
```

Comment:
I would extract this struct and its methods into a separate file called `testing.go`.